index.js

```diff
@@ -15,7 +15,10 @@ const hexRegex3Digit = /^#[a-fA-F0-9]{3}$/;
 const isItNamedColor = (color) => namesAndRGBValues.hasOwnProperty(color);
 
 const shortToFullHex = (hexColor) => {
-  return [...hexColor].map((x, index) => (index != 0 ? x + x : x)).join("");
+  return [...hexColor]
+    .slice(1)
+    .map((x, index) => x + x)
```

Comment:
I like what you did here, and of course you can just remove the index parameter since it is no longer needed.